CHANGELOG.md

```diff
@@ -8,19 +8,20 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Java: Add partial support for `synchronized` blocks in the dataflow IL (#4150)
 
 ### Fixed
-- Constant propagation: In a method call `x.f(y)`, if `x` is a constant then 
```

Comment:
Why the whitespace changes?

Reply:
Autoformatted by my text editor I should add a pre-commit rule